base-theme/layouts/partials/external_resource_link.html

```diff
@@ -13,11 +13,21 @@
 {{- if and $isBroken $backupUrl -}}
   {{- $href = $backupUrl -}}
 {{- end -}}
-{{- partial "link" (dict
+{{- if not (in $href "ocw.mit.edu") -}}
+  {{- partial "link" (dict
       "href" $href
       "text" $text
       "class" $className
       "onClick" $onClick
       "target" "_blank"
-  )
--}}
+    )
+  -}}
+{{- else -}}
+  {{- partial "link" (dict
+    "href" $href
+    "text" $text
+    "class" "text-dark nav-link"
+    "onClick" $onClick
+    )
```

Comment:
I tried all possible combinations of: (OCW link, non OCW link, live-qa link, with non-OCW warning, without non-OCW warning)

It failed in the case of: (OCW link, with non-OCW warning)
Technically it does not make sense to have this use-case in production. However, since non-OCW warning is selected by default, I think we could have cases where Course Authors could accidentally create an OCW link with the warning - in which case the redirect fails to occur, and no warning pop up appears as well.

Current behavior: If you try this in RC, currently a warning appears and the page redirects to OCW link if you click "Continue".

Possible options:

1. Bring back the warning in this specific case (with option to redirect). 
2. Remove warning pop up (even if selected from ocw-studio) and redirect in same tab, if it is truly OCW resource (the behavior from this PR already attempts this. However it does not work because of `"onClick" $onClick` on line 30, as it calls `event.preventDefault()`. Remove line number 30 and it will work.)
3. Set a warning in `ocw-studio` with the same URL filter. Eg.` It looks like the external resource you added is actually OCW. Are you sure you want to enable non-OCW content warning?`

Currently in production we have 1), but I think 2) may be preferable. 
If we do 2), then 3) becomes less important and could be skipped.
If we do 1), then 3) becomes more important.

Reply:
@umar8hassan @ibrahimjaved12 what did you decided to do in this case?